shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-api/src/main/java/org/apache/shardingsphere/data/pipeline/spi/process/JobConfigEventProcess.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.data.pipeline.spi.process;
+
+import org.apache.shardingsphere.infra.util.spi.type.typed.TypedSPI;
+
+/**
+ * Job config event process.
+ */
+public interface JobConfigEventProcess extends TypedSPI {
+    
+    /**
+     * Cleanup pipeline job config.
+     *
+     * @param jobParameter job parameter
+     */
+    void cleanup(String jobParameter);
+    
+    /**
+     * Check job is successful.
+     *
+     * @param jobParameter job parameter
+     * @return true if job is successful otherwise is false
+     */
+    boolean isJobSuccessful(String jobParameter);
+}
```

Comment:
SPI name and methods could be updated, it's too detailed, could to be higher level.